HISTORY.md

```diff
@@ -6,6 +6,7 @@ I tried to give credit whenever possible. If I have missed anyone, kindly add it
 
 ### In Development
 
+- Tweak: Add support for Adafruit trinket3/trinket5/protrinket3/protrinket5 by improved BOARDS_TXT parsing
```

Comment:
can we move this down to where the other Tweak lines are? also credit yourself in brackets (https://github.com/zabereer)